hunting-buddy.lic

```diff
@@ -412,8 +412,13 @@ before_dying do
     stop_script(script_name) if Script.running?(script_name)
   end
   DRCA.release_cyclics(get_settings.cyclic_no_release)
-  Flags.delete('room-check')
-  Flags.delete('familiar_drag')
+  flags_to_delete = [
+    'hunting-buddy-room-check', # there are multiple flags with this prefix, one per friend
```

Comment:
Would it be better to have this be an array of RegExp's?

Reply:
@rpherbig What are you thinking they'd be? I'm not sure they need to be regular expressions, but we could simplify to deleting all flags that start with `hunting-buddy-` prefix.

Reply:
resolved per https://github.com/rpherbig/dr-scripts/pull/5517/commits/297eac9f2e55f865f03342301748158ea58178c4